pkg/kube/pod.go

```diff
@@ -154,6 +154,7 @@ func WaitForPodCompletion(ctx context.Context, cli kubernetes.Interface, namespa
 
 // PodSpecOverride override default pod Spec with the ones provided via specs
 func PodSpecOverride(ctx context.Context, defaultSpecs, overrideSpecs v1.PodSpec) (v1.PodSpec, error) {
+	containers := defaultSpecs.Containers
 	// - Marshal override specs
 	// - Unmarshal override specs on default object so that it overrides only the fields that are present in override specs
```

Comment:
Should this comment be moved below before `Unmarshal`?

Reply:
Done!